src/mlpack/methods/perceptron/perceptron_main.cpp

```diff
@@ -139,9 +152,10 @@ static void mlpackMain()
 
   // If the user isn't going to save the output model or any predictions, we
   // should issue a warning.
-  RequireAtLeastOnePassed({ "output_model", "output" }, false,
+  RequireAtLeastOnePassed({ "output_model", "output", "predictions" }, false,
       "no output will be saved");
-  ReportIgnoredParam({{ "test", false }}, "output");
+  // "output" can be removed in mlpack 4.
```

Comment:
I agree with Marcus, being explicit with `4.0.0` is fine here. :+1:

Reply:
Sorry, forgot to change that